wire-compiler/src/test/java/com/squareup/wire/schema/LinkerTest.kt

```diff
@@ -22,6 +22,7 @@ import com.google.common.jimfs.Jimfs
 import com.squareup.wire.testing.add
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
+import kotlin.test.fail
```

Comment:
Let's use `org.junit.Assert.fail()`. We might have to add the dependency in the module.